server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
 const http = require('http');
 
-// Creating the HTTP server
 const server = http.createServer((req, res) => {
-
-  res.writeHead(200, { 'Content-Type': 'text/plain' });
```

Comment:
The MongoDB URI should be stored in a separate configuration file instead of being hard-coded in the code. This helps with separation of concerns and keeps sensitive information from being exposed in the code.